packages/nextra-theme-docs/src/misc/default.config.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 
 export default {
-  github: 'https://github.com/shuding/nextra',
+  gitLink: 'https://github.com/shuding/nextra',
```

Comment:
How about calling it `projectLink` for more general use? For example it can be linked to any website not necessarily a git hosting platform.

Reply:
Yeah that sounds like a good idea. `projectLink` and `projectLinkIcon`?

Reply:
Yep!